products/health-care/beneficiary-travel/engineering/spikes-for-api-team/api-versioning.md

```diff
@@ -0,0 +1,37 @@
+# Problem: API Versioning
+
+The API versioning schema should be related to the endpoints and functional not the release cycle
+
+## Risks
+
+Updating endpoints without updated functionality can cause unneeded overhead and maintenance for both client and API teams
+
+## Proposed Spike
+
+Iterating on our url versioning strategy to align better with
+
+- best practices
+- Features (that break/change functionality) not release cycles
+
+## Additional Notes
+
+- We can hold at a current/upcoming release until we work through this
+
+## Risks this aims to address
+
+- Too many endpoints being deployed without any meaningful change
+- Leads to confusion on whats the latest spec which endpoint to use and how long will that endpoint be availible
```

Comment:
I would add "Prevent dramatic, frenzied version cutovers by relying on graceful deprecation of older versions"